### DISCUSSION TITLE
My school don't give me email edu

### DISCUSSION THREAD

**Discussion by garpherm** [community/community#120360] [GitHub Education]
### Hi everyone,

i don't have email edu because my school don't give student email edu. Please help me

**Comment by samus-aran** [comment]
Hi @Letuanmp 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
